services/QuillLMS/client/app/bundles/Connect/app.tsx

```diff
@@ -21,14 +21,15 @@ import titleCardActions from './actions/titleCards.ts';
 import quillNormalizer from './libs/quillNormalizer';
 import Home from './components/home.tsx';
 
+import { defaultQueryClientOptions } from '../Shared';
 
 if (process.env.NODE_ENV === 'production') {
   Sentry.init({ dsn: 'https://528794315c61463db7d5181ebc1d51b9@o95148.ingest.sentry.io/210579' })
 }
 
 BackOff();
 const store = createStore();
-const queryClient = new QueryClient()
+const queryClient = new QueryClient({ defaultOptions: defaultQueryClientOptions })
 
```

Comment:
Given the boilerplate of:

`import { defaultQueryClientOptions } from '../Shared';`
`const queryClient = new QueryClient({ defaultOptions: defaultQueryClientOptions })`

I wonder if there is some refactoring opportunity.

`import { DefaultReactQueryClient } from '../Shared';`

and

```

class DefaultReactQueryClient extends QueryClient {
  constructor() {
    super({ defaultOptions: defaultQueryClientOptions })
  }
}
```

That way in the future, there's only one import instead of two and you standardize access to the QueryClient in the future.

Reply:
great suggestion! I'll make this update